examples/vue-app/src/components/HelloWorld.vue

```diff
@@ -214,14 +245,17 @@ const transferSPL = async () => {
     100000,
     6
   );
+  const latestBlockhash = await conn.getLatestBlockhash();
+  // create v0 compatible message
+  const messageV0 = new TransactionMessage({
+    payerKey: new PublicKey(publicKeys![0]),
+    instructions: [transferInstructions],
+    recentBlockhash: latestBlockhash.blockhash,
+  }).compileToV0Message();
+  const transactionV0 = new VersionedTransaction(messageV0);
```

Comment:
Legacy should tested with legacy transaction
should not use VersionTransaction here